service/src/main/resources/message-success-schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://jboss.org/sbomer/message.json",
+  "title": "UMB message",
+  "description": "A UMB message sent after the Sbom generation process is done",
+  "type": "object",
+  "properties": {
+    "purl": {
+      "description": "A package URL to uniquely identify the main component",
+      "type": "string"
+    },
+    "productConfig": {
+      "type": "object",
+      "description": "Product information related to the Sbom and build",
+      "properties": {
+        "errataTool": {
+          "description": "Product information from the Errata Tool",
+          "type": "object",
+          "properties": {
+            "productName": {
+              "description": "Errata product name",
+              "type": "string"
+            },
+            "productVersion": {
```

Comment:
I think I missed this in the Google doc, but I think this should be a Product Variant. At least that is the object that CPEs are defined on in Errata Tool.

Reply:
`productVariant` added.